integrations/crowdnode/src/main/java/org/dash/wallet/integrations/crowdnode/utils/CrowdNodeConstants.kt

```diff
@@ -42,6 +41,12 @@ object CrowdNodeConstants {
     val API_CONFIRMATION_DASH_AMOUNT: Coin = Coin.valueOf(54321)
     val MINIMUM_LEFTOVER_BALANCE: Coin = Coin.valueOf(30000)
 
+    object WithdrawalLimits {
+        val DEFAULT_LIMIT_PER_TX: Coin = Coin.COIN.multiply(15)
+        val DEFAULT_LIMIT_PER_HOUR: Coin = Coin.COIN.multiply(30)
+        val DEFAULT_LIMIT_PER_DAY: Coin = Coin.COIN.multiply(60)
+    }
+
```

Comment:
Should we put a comment here with this link where we can find the current limits?
// https://knowledge.crowdnode.io/en/articles/6387601-api-withdrawal-limits

Reply:
Good idea.